projects/kalidea/kaligraphi/src/lib/02-form/kal-autocomplete/kal-autocomplete.directive.ts

```diff
@@ -129,7 +129,7 @@ export class KalAutocompleteDirective<T = string> implements OnInit, OnDestroy {
       const panelClass = this.theme ? this.theme.kalThemeAsClassNames : [''];
       const config: OverlayConfig = {
         positionStrategy: this.positionsList,
-        scrollStrategy: this.overlay.scrollStrategies.reposition({scrollThrottle: 100}),
+        scrollStrategy: this.overlay.scrollStrategies.close(),
```

Comment:
this fix will be anoying soon, we should later add scrollStrategies @Input for this directive.